test/kubemark/start-kubemark.sh

```diff
@@ -102,6 +102,9 @@ function write-pki-config-to-master {
     sudo bash -c \"echo ${CA_CERT_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/ca.crt\" && \
     sudo bash -c \"echo ${MASTER_CERT_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/server.cert\" && \
     sudo bash -c \"echo ${MASTER_KEY_BASE64} | base64 --decode > /home/kubernetes/k8s_auth_data/server.key\" && \
+    sudo bash -c \"echo ${REQUESTHEADER_CA_CERT} | base64 --decode > /home/kubernetes/k8s_auth_data/aggr_ca.crt\" && \
+    sudo bash -c \"echo ${PROXY_CLIENT_CERT} | base64 --decode > /home/kubernetes/k8s_auth_data/proxy_client.crt\" && \
```

Comment:
Seems like PROXY_CLIENT_CERT is obtained inside `build-kube-master-certs` function - https://github.com/kubernetes/kubernetes/blob/73326ef01d2d7c7eb3b8738ab57bf3b3d268ef97/cluster/common.sh#L580. But we're not using that function when creating the kubemark master. We're instead doing it through `create-master-environment-file` function at the top of this file.

Reply:
Guess replacing it `PROXY_CLIENT_CERT_BASE64` should work.

Reply:
Thanks for pointing this out. I think it may be a bit more complicated. The same would be true of MASTER_KEY_BASE64. I think we get it from parse-master-env. If true however I still messed up the variable name the value should be captured in.

Reply:
`MASTER_KEY_BASE64` is calculated inside `create-certs` function that we're calling before this. And that seems to also calculate the aggregator-related certs.
So fixing the variable names should work.